charts/prometheus-nginx-exporter/ci/ci-values.yaml

```diff
@@ -0,0 +1,16 @@
+---
+nginxServer: http://ci-nginx:8080/stub_status
+nginx:
```

Comment:
Imho it fails because the nginxdefault service is set to type loadbalancer.
I would try like this:

```
nginxServer: http://ci-nginx:8080/stub_status

nginx:
  enabled: true

  fullnameOverride: ci-nginx

  service:
    type: ClusterIP
    ports:
      http: 8080

  serverBlock: |-
    server {
      listen 0.0.0.0:8080;
      root /app;
      location / {
        index index.html;
      }
      location /stub_status {
        stub_status on;
      }
    }

```

Reply:
i've added a commit...